data_steward/cdr_cleaner/cleaning_rules/clean_ppi_numeric_fields_using_parameters.py

```diff
@@ -82,6 +86,51 @@
     WHEN observation_concept_id IN (1585889, 1585890) AND (value_as_number < 0 OR value_as_number > 20) THEN 2000000010
     WHEN observation_concept_id IN (1585795, 1585802, 1585864, 1585870, 1585873, 1586159, 1586162) AND (value_as_number < 0 OR value_as_number > 99) THEN 2000000010
     WHEN observation_concept_id = 1585820 AND (value_as_number < 0 OR value_as_number > 255) THEN 2000000010
+    WHEN observation_source_concept_id IN (1333015, 1585889) AND (value_as_number < 0 OR value_as_number > 20) THEN 2000000013
+  ELSE value_as_concept_id
+END AS
+    value_as_concept_id,
+    qualifier_concept_id,
+    unit_concept_id,
+    provider_id,
+    visit_occurrence_id,
+    observation_source_value,
+    observation_source_concept_id,
+    unit_source_value,
+    qualifier_source_value,
+    value_source_concept_id,
+    value_source_value,
+    questionnaire_response_id
+FROM
+    {{project}}.{{dataset}}.observation""")
+
+HOUSEHOLD_SIZE_SANDBOX_QUERY = jinja_env.from_string("""
+CREATE OR REPLACE TABLE
+    `{{project}}.{{sandbox_dataset}}.{{intermediary_table}}` AS (
+SELECT * 
+FROM
+    `{{project}}.{{dataset}}.observation`
+WHERE
+    (observation_source_concept_id = 1333015 AND (value_as_number < 0 OR value_as_number > 20))
+OR 
+    (observation_source_concept_id = 1585889 AND (value_as_number < 0 OR value_as_number > 20)))
+""")
+
+CLEAN_HOUSEHOLD_SIZE_QUERY = jinja_env.from_string("""SELECT
+    observation_id,
+    person_id,
+    observation_concept_id,
+    observation_date,
+    observation_datetime,
+    observation_type_concept_id,
+CASE
```

Comment:
I will suggest adding the conditions in the existing query, so we can run only 2 queries instead of 4 queries. 
You can add the new ticket number at the end. you can still use the 827 as the main ticket number.